emcc.py

```diff
@@ -60,7 +60,7 @@
 CXX_ENDINGS = ('.cpp', '.cxx', '.cc', '.c++', '.CPP', '.CXX', '.CC', '.C++', '.ii')
 OBJC_ENDINGS = ('.m', '.mi')
 OBJCXX_ENDINGS = ('.mm', '.mii')
-SPECIAL_ENDINGLESS_FILENAMES = ('/dev/null',)
+SPECIAL_ENDINGLESS_FILENAMES = ('/dev/null' if not WINDOWS else 'NUL',)
```

Comment:
`'NUL' if WINDOWS else '/dev/null`)` to avoid negatives?

Reply:
Hmm, I agree a negative is not great, but on the other hand keeping the most-likely option in the first position is also clearer, I think.

Perhaps we could define `POSIX = not WINDOWS`? Not sure if it's worth it.